vendir.yml

```diff
@@ -8,7 +8,7 @@ directories:
   - path: .
     git:
       url: https://github.com/cloudfoundry/capi-k8s-release
-      ref: bb40b9d67db1c3d150791d773752d1d788cdef91
+      ref: rename-kpack-watcher-to-controllers
```

Comment:
@cwlbraa @ericpromislow FYI using a branch causes issues for us. `vendir sync` gets out of date as the HEAD of the branch changes. I've switched to using the sha directly `6645721`.

build failure: https://release-integration.ci.cf-app.com/teams/main/pipelines/cf-for-k8s/jobs/test-vendir-sync-on-cf-for-k8s/builds/269